plugins/mongodb/mongo_collection_

```diff
@@ -94,25 +94,25 @@ def getCollstats(graphtype):
                 con = pymongo.MongoClient(settings_host, int(settings_port))
 
         if settings_user:
-                db = con['admin']
+                db = con[settings_db]
                 db.authenticate(settings_user, settings_password)
 
         stats_tmp = {}
-        for dbname in con.database_names():
+        for dbname in con.list_database_names():
                 if dbname in settings_ignoredb:
                     continue
                 db = con[dbname]
-                for coll in db.collection_names():
+                for coll in db.list_collection_names():
                         if coll.startswith('system.'):
                                 continue
                         stats = db.command("collstats", coll)
                         collname = dbname + "_" + coll.replace('.', '_')
-                        if not stats_tmp.has_key(collname):
+                        if not collname in stats_tmp:
```

Comment:
`flake8` advises to use `... not in ...` instead of `not ... in ...` (avoiding the ambiguity of precedence).